### DISCUSSION TITLE
Support for the Gamesir T1s

### DISCUSSION THREAD

**Discussion by leocruga** [ndeadly/MissionControl#182] [Ideas]
Great developers! Would it be possible to add support for the Gamesir T1s Control? it would be of great value, thank you very much!

I try to connect to the Switch, but it appears that it is connected, but nothing works!

**Comment by ndeadly** [comment]
If it's connecting to the switch it will work, I just need to create a handler for it. Could you re-open this as a controller request under the Issues section? I'll need a screenshot of the output of btdb.nro for the hardware ID too.

**Reply by leocruga** [reply]
Certainly, without any problem, I will create the problem in the corresponding section, the only problem is that I am a layman so I don't know how to capture this screen you asked for, but I will try to inform myself, thank you very much for answering!
